modules/nf-core/bbmap/dedupe/main.nf

```diff
@@ -0,0 +1,39 @@
+process BBMAP_DEDUPE {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
This is a problem here, as the `base.config` setting will ask for 36GB of memory which will cause Nextflow to quit. I have been changing modules to `process_macbook` for now, which creates a more reasonable limit of 9 cores and 15 GB of memory.